google-account-plugin/src/com/google/cloud/tools/intellij/stats/UsageTracker.java

```diff
@@ -33,4 +34,26 @@ void trackEvent(@NotNull String eventCategory,
       @Nullable String eventLabel,
       @Nullable Integer eventValue);
 
+  PartialTrackingEventAction trackEvent(String category);
+
+  interface SendsEvent {
+
+    void send();
+  }
+
+
+  interface PartialTrackingEventAction extends SendsEvent {
+
+    PartialTrackingEventLabel withAction(String value);
+
+    interface PartialTrackingEventLabel extends SendsEvent {
+
+      PartialTrackingEventValue andLabel(String value);
+
+      interface PartialTrackingEventValue extends SendsEvent {
```

Comment:
Yea that makes sense. I guess it will always be the case that this order should be enforced.